Cisco/cisco-ise/tests/test_conf_events.json

```
@@ -0,0 +1,38 @@
+{
+  "input": {
+    "sekoiaio": {
+      "intake": {
+        "dialect": "Cisco ISE",
+        "dialect_uuid": "8a9894f8-d7bc-4c06-b96a-8808b3c6cade"
+      }
+    },
+    "message": "INFO: Configuration Changed: Admin=john.doe; Object Type=EPPurgeScheduler; Object Name=f36afcff-e3af-4a70-99c0-5e5304c1c336"
+  },
+  "expected": {
+    "message": "INFO: Configuration Changed: Admin=john.doe; Object Type=EPPurgeScheduler; Object Name=f36afcff-e3af-4a70-99c0-5e5304c1c336",
+    "event": {
+      "kind": "event",
+      "type": ["info"],
```

The ECS specification allows us defining finer [`event.type`](https://www.elastic.co/guide/en/ecs/8.6/ecs-allowed-values-event-category.html#ecs-event-category-configuration) for a configuration event.
```suggestion
      "type": ["change"],
```
Could you also add test for a  configuration creation